### ISSUE TITLE
[EquipmentAndQuickSlots] Upgrading Armor while inventory is full makes the item disappear.

### ISSUE THREAD

**Issue by virtualbjorn** [RandyKnapp/ValheimMods#90]
I was upgrading my troll armor in a hurry while my inventory is full and when I check if I could equipped them after the upgrade, they're all gone.

**Comment by RandyKnapp** [comment]
I'll look into this right away

**Comment by virtualbjorn** [comment]
Thanks for a really good mod, just some bumps but overall its good. @RandyKnapp

**Comment by the-madd-hatter** [comment]
Suspect the same thing happened when I recovered equipment from tombstone, had a full set of backup gear equipped and the Mod allowed me to get all items from the tombstone even though it overflowed my inventory.

I realized that the 2 pieces of gear that were missing were not lost as at the forge they showed up in the upgrade list. Removing all items from inventory into chests had my weight at 18/300.

Installed a simple extended inventory mod and the 2 items that were 'missing' were on the next row down from the initial inventory limit.

Possible override of inventory actions is not accounting for the inventory space check (prevents you from upgrading an item with full inventory, also prevents you from auto loot all on tombstone when you don't have enough slots in inventory).
